src/components/layout/Panel.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ComponentChildren, JSX } from 'preact';
+
+import type { IconComponent, CompositeProps } from '../../types';
+import { downcastRef } from '../../util/typing';
+
+import Card from './Card';
+import CardContent from './CardContent';
+import CardHeader from './CardHeader';
+import CardTitle from './CardTitle';
+import CardActions from './CardActions';
+
+import Scroll from '../data/Scroll';
```

Comment:
Oops this import is out of order. I'll fix along with other feedback on this PR.

Reply:
Maybe we should consider adding a linting rule that helps us keep the imports the way we want? Being able to fix this kind of things with `make format` would be very convenient.